pull-requests.md

```diff
@@ -31,26 +41,19 @@ If anyone has engaged in behavior that is potentially illegal or makes you or so
-
-Make sure the issue isn't assigned and no one has already opened a pull request for the task. You can do that by checking the issue and searching open pull requests
-
-If you're not sure where to start, reach out to us on [Discord]() and ask for feedback before starting on any work.
-
-### Opening a pull request
-
 When you're ready to open a pull request, follow these guidelines:
 
-- **PRs should be short and focused to one issue:** If possible, group related set of changes into multiple, small PRs. If a PR becomes too large, the reviewer may close it and ask that you to submit pull requests for each logical set of changes that belong together.
-- **Add a description of what the pull request is doing:** Provide as much context and reason for opening the pull request.
-- **Add the link to the issue you are closing or working on:** Add 'Fixes' or 'Relates to' and the related issue in the description of the pull request. See [github link here]()
-- **Accompany code example changes with content changes:** This is important to ensure that updated examples are served correctly. If content changes also affect how examples are used, the related code examples should be updated in associated repositories.
-- **Add 'depends on'** with a link to a dependency if there are PRs that must land first (e.g. linked examples in other repos).
-- **Add a reviewer** if you know who should review your PR already, such as a team member or a topic owner.
+- **Add the link to the issue you are closing:** Add 'Fixes' or 'Relates to' and the related issue in the description of the pull request.
+  More information about issues can be found [in our Community guidelines pages](https://developer.mozilla.org/en-US/docs/MDN/Community/Issues).
+- **Add a description of the changes:** Provide as much context and rationale for the pull request as possible.
+- **PRs should be short and focused to one issue:** If possible, group related set of changes into multiple, small PRs.
+  If a PR becomes too large, the reviewer may close it and ask that you to submit pull requests for each logical set of changes that belong together.
 - **Don't make grammar-only changes.**
   MDN Web Docs contains technical documentation; you should not suggest prose style changes except where grammar is incorrect.
+- **Accompany code example changes with content changes:** This is important to ensure that updated examples are served correctly.
+  If you're making content changes that affect how examples are used, the related code examples should also be updated.
+- **Add 'depends on'** with a link to a dependency if there are PRs that must land first (e.g., code examples in other repositories).
```

Comment:
This could maybe combined with the previous point on "Add the link to the issue you are closing"

Reply:
Yeah true. Although having it as a separate point makes it also stand out, what do you think? I'm happy with either.

Reply:
Hmm, that is a good point. We can list them next to each other in that case.

Reply:
done in d964768322d5a3c78a6961ca9d1266db6df587f4